dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,22 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recovering from Failure when Expanding Volumes
+
+If expanding underlying storage fails either on master or node,
+{product-title} admin can manually recover the PVC state and cancel the resize requests that are
+continuously retried by the controller without administrator intervention.
+
+Currently, this can be done manually by completing the following steps:
+
+1. Mark the PV that is bound to the claim (PVC) with the `Retain` reclaim policy.
+   This can be done by editing the PV and changing persistentVolumeReclaimPolicy to `Retain`.
```

Comment:
If the PV's reclaim policy is not "Retain", first changing the reclaim policy to "Retain"?

Reply:
Isn't that already evident that, admin needs to ensure that PV must have `Retain` policy? I do not think we need to reword this.

Reply:
What is `administrator` and what is `admin`? I think cluster administrator is needed to edit PVs.